.github/workflows/create-or-update-release-pr.yml

```diff
@@ -13,9 +13,10 @@ on:
         type: choice
         description: Release Scope
         options:
-          - experimental
-          - sdk
-          - all
+          - experimental # all packages in experimental/packages
+          - sdk # all SDK packages, experimental and stable, excluding semantic conventions
+          - semconv # only semantic convention package
+          - all # all release packages, including API, excluding semconv
```

Comment:
Added clarity for what's included for each option. "All" is now somewhat misleading since semconv isn't in there, but I figured that would make this big change even bigger and figured we could change "all" to something else later. Semconv should never be released on a schedule with anything else other than semconv versioning.